lib/auth/password.go

```diff
@@ -132,10 +117,8 @@ func (s *Server) ChangePassword(req services.ChangePasswordReq) error {
 			Password: req.OldPassword,
 		}
 	}
-	if req.U2FSignResponse != nil {
-		authReq.U2F = &U2FSignResponseCreds{
-			SignResponse: *req.U2FSignResponse,
-		}
+	if req.WebauthnResponse != nil {
+		authReq.Webauthn = req.WebauthnResponse
```

Comment:
i was questioning how this was working before :laughing:, but this is already set in line 113, remove one of them?

Reply:
That explains my confusion too. Done.